frontend/src/features/public-form/components/FormFields/FormFieldsContainer.tsx

```diff
@@ -58,6 +58,7 @@ export const FormFieldsContainer = ({
         <PublicSwitchEnvMessage
           responseMode={form?.responseMode}
           isAuthRequired={isAuthRequired}
+          authType={form?.authType}
```

Comment:
`form` should be defined here right, since there's a check on line 31?

Reply:
Looks like the form check only affects the `renderFields` memo, so it's not guaranteed to be defined here (curse indentation! 😜)